content/efcore/getting-started/loading-related-data.md

```diff
@@ -24,67 +24,67 @@ using (var context = new LibraryContext())
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var book = context.Books 
-		.Include(b => b.Editions) 
+		.Include(b => b.CheckoutRecords) 
 		.Include(b => b.Author)
-		.Single(b => b.BookId == 1); 
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=multiple-include-methods} 
 :::
  
 ## Multi-layer Inclusion 
  
-Now, what happens if one of the properties we include also has non-primitive type properties? By default, they won't be loaded from the database, but we can tell EF Core to load them using the `ThenInclude` method. For example, we can load the publisher information for the editions of a book like so: 
+Now, what happens if one of the properties we include also has non-primitive type properties? By default, they won't be loaded from the database, but we can tell EF Core to load them using the `ThenInclude` method. For example, we can load the reader information for the checkout records of a book like so: 
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var book = context.Book 
-		.Include(b => b.Editions) 
-			.ThenInclude(e => e.Publisher)
-		.Single(b => b.BookId == 1) ; 
+		.Include(b => b.CheckoutRecords) 
+			.ThenInclude(cr => cr.Reader)
+		.Single(b => b.Id == 1); 
 } 
 ``` 
 :::repl{data-name=then-include-method} 
 :::
  
-We can also chain `ThenInclude` calls to include deeper layers of related data. For example, if we want to find out the publisher information for the different editions of Fredrick Douglass' books in this library, we will need to chain `ThenInclude` calls to get the book, edition, and publisher information. 
+We can also chain `ThenInclude` calls to include deeper layers of related data. For example, if we want to find out the reader information for the checkout records of Fredrick Douglass' books in this library, we will need to chain `ThenInclude` calls to get the book, checkout records, and reader information. 
  
 ```{.snippet} 
 using (var context = new LibraryContext()) 
 { 
 	var author = context.Authors 
 		.Include(a => a.Books) 
-			.ThenInclude(b => b.Editions) 
-				.ThenInclude(e => e.Publisher)
+			.ThenInclude(b => b.CheckoutRecords) 
+				.ThenInclude(cr => cr.Reader)
 		.Single(a => a.LastName == "Douglass"); 
 } 
 ``` 
 :::repl{data-name=then-include-method-chain} 
 :::
  
-A mix of `Include` and `ThenInclude` commands can also be chained together to include related data from multiple layers across related entities. In the following example, we are looking for the author, editions, and edition publishers of _The Call of the Wild_. 
+A mix of `Include` and `ThenInclude` commands can also be chained together to include related data from multiple layers across related entities. In the following example, we are looking for the author, checkout records, and the readers of those records of _Murder on the Orient Express_. 
```

Comment:
In the following example, we chain together `Include` and `ThenInclude` commands to load the author of _Murder on the Orient Express_, as well as all of the readers who have checked it out.